src/EnergyPlus/FileSystem.cc

```diff
@@ -134,10 +131,10 @@ namespace FileSystem {
             if (pathTail.size() == 0)
                 return absoluteParentPath;
             else
-                return absoluteParentPath + pathChar + pathTail;
+                return absoluteParentPath + DataStringGlobals::pathChar + pathTail;
 
         } else {
-            DisplayString("ERROR: Could not resolve path for " + path + ".");
+            std::cout << "ERROR: Could not resolve path for " + path + "." << std::endl;
```

Comment:
Unexpected, but OK.

Reply:
This was to avoid the `#include <EnergyPlus/DisplayRoutines.hh>` include since that has `#include <EnergyPlus/EnergyPlus.hh>`, which then requires ObjexxFCL includes. I could look to see if `DisplayRoutines.hh` ACTUALLY requires `EnergyPlus.hh` header or not.

Reply:
I forgot that `DisplayRoutines.cc` has the following headers as well.
```cpp
// ObjexxFCL Headers
#include <ObjexxFCL/Fmath.hh>

// EnergyPlus Headers
#include <EnergyPlus/DataGlobals.hh>
#include <EnergyPlus/DataSystemVariables.hh>
#include <EnergyPlus/DisplayRoutines.hh>
```

Reply:
That's what I figured.  I wasn't expecting it when I saw it, but I figured it was to avoid a dependency. :+1: